src/pages/index.tsx

```diff
@@ -104,7 +103,7 @@ function Page(props: Props) {
   )
 }
 
-export const query = graphql`
+export const querySSG = graphql`
```

Comment:
How can we know if the query runs at SSG or not?

Reply:
if it has `graphql` it runs on SSG. If it has a `gql` it runs on SSR or client side